### DISCUSSION TITLE
大佬们问一下为什么提纳里的天赋无法提升13级，是满命的

### DISCUSSION THREAD

**Discussion by sans23456** [Melledy/Grasscutter#1732] [General]
![原神 ](https://user-images.githubusercontent.com/82899463/187825873-3eb3ea6f-c5a3-4dc8-9ea8-ce66ab2e6178.png)

**Comment by ghost** [comment]
Your binoutput files (the ones in `resources/BinOutput/Talent/AvatarTalents`) need to be updated.

**Reply by sans23456** [reply]
我是最新的resources
